composer.json

```diff
@@ -23,7 +23,6 @@
         "psr/event-dispatcher": "^1.0",
         "psr/http-factory": "^1.0",
         "psr/log": "^1.1",
-        "yiisoft/access": "^1.0",
```

Comment:
Previously you were able to do

```php
if ($currentUser->can('update_post', $post)) {
   $post->setModifiedBy($currentUser);
   $postRepository->persist($post);
}
```

How to do it now?

Reply:
Returned the `can` method.